xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleCreatePHPDisbursement.java

```diff
@@ -0,0 +1,123 @@
+
+                        /**
+                         * Second option. Create with individual value of required params.
+                         */
+                        DisbursementPHP disbursement2 = DisbursementPHP.createPHPDisbursement(
+                                        "xendit_idempotency_key".concat(new Date().toString()), "reference_id_value",
+                                        "PHP",
+                                        disbursementChannel.getChannelCode(), "John etc", "123456",
+                                        "Disbursement description", 50000);
+                        System.out.print(disbursement2);
+                        /**
+                         * Third option. Create with individual value of required params + optional
+                         * ReceiptNotification.
+                         */
+                        ReceiptNotification receiptNotification = ReceiptNotification.builder()
+                                        .emailTo(new String[] { "test@emailTo.com" })
+                                        .emailCC(new String[] { "test@emailCC.com" })
+                                        .emailBcc(new String[] { "test@emailBcc.com" })
+                                        .build();
+                        DisbursementPHP disbursement3 = DisbursementPHP.createPHPDisbursement(
+                                        "xendit_idempotency_key".concat(new Date().toString()), "reference_id_value",
+                                        "PHP",
+                                        disbursementChannel.getChannelCode(), "John etc", "123456",
+                                        "Disbursement description", 50000, receiptNotification);
+                        System.out.print(disbursement3);
+                        /**
+                         * Fourth option. Create with individual value of required params + optional
+                         * Beneficiary.
+                         */
+                        Beneficiary beneficiary = Beneficiary.builder()
+                                        .type("INDIVIDUAL")
+                                        .givenNames("Test Name")
+                                        .middleName("Middle Name")
+                                        .surname("Sur Name")
+                                        .businessName("Test")
+                                        .streetLine1("Jl. 123")
+                                        .streetLine2("Jl. 456")
+                                        .city("Jakarta Selatan")
+                                        .province("DKI Jakarta")
+                                        .state("Test")
+                                        .country("Test")
+                                        .zipCode("12345")
+                                        .mobileNumber("9876543210")
+                                        .phoneNumber("987654321")
```

Comment:
can add sample country code for this? `+63`

Reply:
72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f